.circleci/config.yml

```diff
@@ -338,6 +365,7 @@ workflows:
   version: 2
   calypso:
     jobs:
+      - build-notifications
```

Comment:
One way we could avoid wasting container time is to filter branches. If we know that notifications work happens in branches like: `notifications/make-notifications-great`, we could add a filter here so these aren't being built for PRs that don't require them.

Reply:
That sounds fantastic! It would have been nicer to go by `notifications` label, but that doesn't look possible. Considering we usually preface branch names with an action (eg. `fix/`, `add/`), maybe just drop the caret?